### PR TITLE
[Fix]: add convert script for Votenet

### PR DESCRIPTION
**Pull request by encore-zhou**
Fix issue #145 by  converting the keys in the checkpoints of votenet models.

### CODE REVIEW

File: tools/convert_votenet_checkpoints.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        ('bbox_head.conv_pred.conv_out.weight', [(2, -NUM_CLASSES)]),
+        'bbox_head.conv_pred.conv_reg.bias':
+        ('bbox_head.conv_pred.conv_out.bias', [(2, -NUM_CLASSES)])
+    }
+
+    RENAME_KEYS = {
+        'bbox_head.conv_pred.shared_convs.layer0.conv.weight':
+        'bbox_head.conv_pred.0.conv.weight',
+        'bbox_head.conv_pred.shared_convs.layer0.conv.bias':
+        'bbox_head.conv_pred.0.conv.bias',
```
Comment (ZwwWayne) [inline tools/convert_votenet_checkpoints.py:48]: We can directly replace `bbox_head.conv_pred` by `bbox_head.conv_pred.shared_convs.layer`?

File: tools/convert_votenet_checkpoints.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+from mmcv import Config
+from mmcv.runner import load_state_dict
+
+from mmdet3d.models import build_detector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='MMDet test (and eval) a model')
+    parser.add_argument('config', help='test config file path')
```
Comment (ZwwWayne) [inline tools/convert_votenet_checkpoints.py:12]: The checkpoint should contain config in meta, so maybe we can directly use the config in checkpoint without loading config.

File: tools/convert_votenet_checkpoints.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
+import argparse
```
Comment (ZwwWayne) [inline tools/convert_votenet_checkpoints.py:1]: Add docstring in the header to indicate the usage of this script.

**Review by ZwwWayne — APPROVED** [review]

**Comment by EricWiener** [comment]
This is still an issue. When I run `python tools/test.py configs/votenet/votenet_16x8_sunrgbd-3d-10class.py checkpoints/votenet_16x8_sunrgbd-3d-10class_20200620_230238-4483c0c0.pth --show --show-dir ./data/votenet/show_results`, I get the message:

```
/home/eric/miniconda3/envs/open-mmlab/lib/python3.7/site-packages/mmcv/cnn/bricks/conv_module.py:107: UserWarning: ConvModule has norm and bias at the same time
  warnings.warn('ConvModule has norm and bias at the same time')
The model and loaded state dict do not match exactly

unexpected key in source state_dict: bbox_head.conv_pred.0.conv.weight, bbox_head.conv_pred.0.conv.bias, bbox_head.conv_pred.0.bn.weight, bbox_head.conv_pred.0.bn.bias, bbox_head.conv_pred.0.bn.running_mean, bbox_head.conv_pred.0.bn.running_var, bbox_head.conv_pred.0.bn.num_batches_tracked, bbox_head.conv_pred.1.conv.weight, bbox_head.conv_pred.1.conv.bias, bbox_head.conv_pred.1.bn.weight, bbox_head.conv_pred.1.bn.bias, bbox_head.conv_pred.1.bn.running_mean, bbox_head.conv_pred.1.bn.running_var, bbox_head.conv_pred.1.bn.num_batches_tracked, bbox_head.conv_pred.conv_out.weight, bbox_head.conv_pred.conv_out.bias

missing keys in source state_dict: bbox_head.conv_pred.shared_convs.layer0.conv.weight, bbox_head.conv_pred.shared_convs.layer0.conv.bias, bbox_head.conv_pred.shared_convs.layer0.bn.weight, bbox_head.conv_pred.shared_convs.layer0.bn.bias, bbox_head.conv_pred.shared_convs.layer0.bn.running_mean, bbox_head.conv_pred.shared_convs.layer0.bn.running_var, bbox_head.conv_pred.shared_convs.layer1.conv.weight, bbox_head.conv_pred.shared_convs.layer1.conv.bias, bbox_head.conv_pred.shared_convs.layer1.bn.weight, bbox_head.conv_pred.shared_convs.layer1.bn.bias, bbox_head.conv_pred.shared_convs.layer1.bn.running_mean, bbox_head.conv_pred.shared_convs.layer1.bn.running_var, bbox_head.conv_pred.conv_cls.weight, bbox_head.conv_pred.conv_cls.bias, bbox_head.conv_pred.conv_reg.weight, bbox_head.conv_pred.conv_reg.bias

```

**Pull request merged**